.github/workflows/sonar_branch_analysis.yml

```diff
@@ -6,7 +6,7 @@ on:
     branches:
       - main
       - candlepin-*-HOTFIX
-      - feature/autoreg_v2
+      - feature/*
```

Comment:
Currently this workflow isn't running on the feature/autoreg_v2 branch, because we need to add this line in that branch too